Resources/public/js/views/ez-studiopresentationview.js

```diff
@@ -19,7 +19,14 @@ YUI.add('ez-studiopresentationview', function (Y) {
      * @constructor
      * @extends eZ.TemplateBasedView
      */
-    Y.eZ.StudioPresentationView = Y.Base.create('studioPresentationView', Y.eZ.TemplateBasedView, [], {
+    Y.eZ.StudioPresentationView = Y.Base.create('studioPresentationView', Y.eZ.TemplateBasedView, [Y.eZ.HeightFit], {
+
+        initializer: function () {
+            this.after('activeChange', function () {
+                this.get('container').one('.ez-studiopresentation-content').set('src', this.get('iframeSource'));
```

Comment:
I would put that in a dedicated method call `_setIFrameSource` so that you can write:
```js
this.after('activeChange', this._setIframeSource);
```
which is way more readable.